docs/aws-lambda-extension.asciidoc

```diff
@@ -98,13 +98,34 @@ For a _minimal configuration_ you need to specify the `ELASTIC_APM_LAMBDA_APM_SE
 [[aws-lambda-handler]]
 === Configuring the Agent and Lambda Function handler
 
-Once you've installed the extension, there's one last step to take. You'll need to wrap the Lambda function handler.
+Once you've installed the extension, there's one last step to take. You'll need to install the correct language agent and wrap the Lambda function handler.
+
 
 [discrete]
 [[aws-lambda-nodejs]]
 ==== Node.js
 
-In Node.js, you wrap a Lambda function handler using the following syntax.
+To install the Node.js agent in Lambda you'll use a second layer.  You'll need to https://github.com/elastic/apm-agent-nodejs/releases[pick the right ARN from the agent release table]. The ARN has the pattern `arn:aws:lambda:<AWS_REGION_KEY>:267093732750:layer:elastic-apm-node-ver-<APM_EXTENSION_VERSION>-0:<LAYER_VERSION>` and depends on:
+
+* The AWS region your Lambda function runs in. The APM Lambda Extension layer needs to be in the same region as your Lambda function.
+* The version of the APM Lambda Extension you would like to use.
+
+You'll then need to configure your function to use that layer. To add a layer
+
+1. Navigate to your function in the AWS Console
+2. Scroll to the Layers section and click the _Add a layer_ button image:images/config-layer.png[image of layer configuration section in AWS Console]
+3. Choose the _Specify an ARN_ radio button
+4. Enter the Version ARN of the APM Lambda Extension layer in the _Specify an ARN_ text input
+5. Click the _Add_ button
```

Comment:
> From a tech writing/elastic standards point of view, do you think we need to repeat the "how to add a layer in the UI" instructions a second time (as its written now), or can we assume the user will have picked this up from the initial section?

This is hard for me to answer without a solid understanding of AWS Lambda. If I'm understanding correctly, this section of text is duplicated from above, and the Node.js agent is the only agent that needs to run a second layer? My gut is to say yes, it makes sense to repeat these instructions. If they're always going to be the same as the instructions above, we can single-source them by "tagging" the text above and including it down here. That will help with maintenance.

Reply:
✅